### DISCUSSION TITLE
Proposed slot-prop.  Determine if q-expansion-item is expanded.

### DISCUSSION THREAD

**Discussion by rfox12** [quasarframework/quasar#13915] [Ideas / Proposals] [answered]
Within a q-expansion-item header / body I sometimes need to access expansion state to change styles within the item.

Today it looks like the only way to get that state is to manage it myself--carrying around a v-model value... this can be tedious because I generally don't care about that state.  Especially for a long accordion of items.

Wouldn't it be really really easy for Quasar developers to add a slot-prop to both default and header to tell me if the current expansion item is expanded?

**Comment by rfox12** [comment]
I believe this would a one-file change to Quasar... https://github.com/quasarframework/quasar/blob/dev/ui/src/components/expansion-item/QExpansionItem.js

With some guidance I may be able to create a PR.. just not used to using `render()` functions....  
One hint here: https://stackoverflow.com/a/67425504/5153063

**Comment by rstoenescu** [comment, marked as answer]
Slot param "expanded" will be available in Quasar v2.7.6 and v1.19.5
